js/src/launchAsync.ts

```diff
@@ -274,6 +276,24 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             src += '&omkt=' + options.uiLang;
         }
 
+        // The default value for disableLanguageDetection is true, and is used too if option is missed.
```

Comment:
disableLanguageDetection option should default to false.

Would only add the querystring if the option is set to true

Reply:
yes, make sense, was thinking it as default true